### ISSUE TITLE
About the best model

### ISSUE THREAD

**Issue by chenglong521** [Hope1337/YOWOv3#36]
How does your code determine which round of model is the best model and then output the mAp

**Comment by Hope1337** [comment]
@chenglong521 At first, I just eval it at the end of each epoch.

**Comment by chenglong521** [comment]
But the highest mAp you have for each round of evaluation is not necessarily the best model, it may overfit, and it may not work well on the validation set

**Comment by Hope1337** [comment]
@chenglong521 Actually, we can not know which model is the best, just evaluate and choose it by intuition. I follow the simplest way to do that, just evaluate it and the end of each epoch. You can find yourself a better way.
